api/src/test/java/org/pmiops/workbench/billing/FreeTierBillingServiceTest.java

```diff
@@ -455,4 +454,14 @@ private DbWorkspace createWorkspace(
     workspace.setBillingMigrationStatusEnum(billingMigrationStatus);
     return workspaceDao.save(workspace);
   }
+
+  private void assertCloseEnough(double actualValue, double expectedValue) {
```

Comment:
I thought that these were provided methods at first, because they had the same `assert` prefix. I think it would be good to make it clear this is code that we own.

Reply:
I hear what you're saying... there are certainly a huge number of functions starting with assert.

In the present case, though, it's a private method, so there's no risk of using it outside this class. Anyone interested can go to its definition. But I renamed it `assertWithinBillingTolerance()` for now so it's obvious at a glance.